Assets/LeapMotion/Core/Scripts/DataStructures/Pool.cs

```diff
@@ -72,11 +74,7 @@ public static class Pool<T> where T : new() {
       if (_pool == null) _pool = new Stack<T>();
 
       T value;
-      if (_pool.Count > 0) {
-        value = _pool.Pop();
-      } else {
-        value = new T();
-      }
+      value = spawnValue();
```

Comment:
F.

because I'm a derp that's why. spawnValue() should be removed and this should be replaced with the actual pooling logic